### ISSUE TITLE
PDF Manual

### ISSUE THREAD

**Issue by jakkos98** [NikolaiVChr/f16#495]
Hello,
Since I prefer the manuals as one document, I have created one.

The current version includes:

1. list of authors divided into two categories
- "Model by" lists all authors from authors.txt [as of 07/14/2022]
- "Manual by" listing the last editor of each wiki page (https://github.com/NikolaiVChr/f16/wiki) [as of 07/14/2022] plus adding myself to the list
2. content of the README.md file [as of 07/14/2022]
3. Content of all 33 wiki pages (https://github.com/NikolaiVChr/f16/wiki) [as of 07/14/2022]
4. "Tags" from kneeboard examples [as of 07/14/2022]
5. Description of the training scenario levels from the sim training scenario menu [as of 07/15/2022, F-16 from the OPRF hangar]
6. List of RWR symbols from the sim help menu [as of 07/15/2022, F-16 from the OPRF hangar]
7. Checklists from the sim checklist menu [as of 07/15/2022, F-16 from the OPRF hangar]

Future plans:

1. Liveries list for each version (possibly with preview)
2. List of available payloads for each pylon for each version
3. List of payload presets for each version
4. Cockpit overview (similar to Viggen pdf manuals)
5. Keyboard shortcuts reference card

Below I am attaching a PDF and an editable ODT file.

Best regards
Pozdrawiam

Jakub

[F16_Manual.pdf](https://github.com/NikolaiVChr/f16/files/9120367/F16_Manual.pdf)
[F16_Manual.odt](https://github.com/NikolaiVChr/f16/files/9120369/F16_Manual.odt)

**Comment by NikolaiVChr** [comment]
That is awesome done Jakub!

Feel free to add it to the wiki, just make sure you add a note about its date, in case it gets out of date.

**Comment by NikolaiVChr** [comment]
@jakkos98 we had a hostile edit in the wiki, where a link was changed.
That hostile edit is inside your pdf also.
Therefore we have temporarily removed your pdf, until (or if) you remake it.

**Comment by jakkos98** [comment]
@NikolaiVChr Here is pdf. It is the same version as last I posted here. I can upload also editable version if you want.
[F16_Manual.pdf](https://github.com/NikolaiVChr/f16/files/10248239/F16_Manual.pdf)

**Comment by NikolaiVChr** [comment]
You don't understand. The malware link is also inside your pdf.
You have to remake at least the page with the install instructions.

**Comment by jakkos98** [comment]
Yeah, I understood as someone has edited pdf and change it on wiki as I initially posted editable file too. Below is pdf with correct link on page with install instruction.
[F16_Manual.pdf](https://github.com/NikolaiVChr/f16/files/10251973/F16_Manual.pdf)

**Comment by NikolaiVChr** [comment]
Thank you
